src/components/Pool/LoadingList/LoadingRow/LoadingRow.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import Skeleton from 'react-loading-skeleton'
+import { Flex } from 'rebass'
+import DoubleCurrencyLogo from '../../../DoubleLogo'
+import { Row } from './LoadingRow.styles'
+
+export function LoadingRow() {
+  return (
+    <Row>
+      <Flex alignItems="center" flexDirection="row">
+        <DoubleCurrencyLogo spaceBetween={0} marginLeft={0} marginRight={14} top={0} loading size={45} />
+        <Flex justifyContent="flex-start" flexDirection="column">
+          <Skeleton height="20px" width="35px" />
+
+          <Skeleton height="20px" width="48px" />
+        </Flex>
+      </Flex>
+      <Flex alignItems="center">
+        <Skeleton height="15px" width="66px" />
+        <Skeleton style={{ marginLeft: '4px' }} height="15px" width="65px" />
+      </Flex>
+      <Flex alignItems="center">
+        <Skeleton height="17px" width="91px" />
+      </Flex>
+      <Flex alignItems="center">
+        <Skeleton height="17px" width="91px" />
+      </Flex>
+      <Flex alignItems="center">
+        <Skeleton height="22px" width="48px" />
+      </Flex>
+    </Row>
+  )
+}
```

Comment:
I think the biggest thing I don't like about using this is that in 90% of the cases components are really small as they should be and it will just clutter our codebase with unnecessary extra files with small amount of code like in this case
- In most cases there is only one type and when developing I think Its much more convenient to have type right above export.
- And regarding style I think maybe just extra style folder would work or moving styles to bottom part of component is also option...

Reply:
I think we should find our sweet spot in this regard. I understand that having a lot of files is bad but having huge files, with a lot of code is even badder IMO. Big files are prone to bad practices and more code fatigue. Searching for a file is simpler than scrolling a file and make your eyes search for the desired code. Also, I think different files make it easier for review.

@Mi-Lan to answer your bullet points, I believe that we should define a solution for worst case scenarios and apply it to simpler cases too. I know that's kind of a burden but I prefer consistency rather than huge files.

Loving this discussion BTW!

Reply:
- I would encourage you to go through codebase and see where this pattern will be helpful, I did and don't think it would solve that issue for overbloated components
- I would say that for the most over bloated components its not because of styles or types but because of components thats poorly structured and that breaking it down into smaller components with separate file for styles would solve this issue much more efficiently. It maybe wouldn't account for couple of overbloated components but would solve issues in 99% and because of those exceptions I don't think we should go this route.

I think because we don't have a clear agreement we should revert to how it was until we agree because having inconsistent component pattern is big no no. 
@berteotti @denciu

Reply:
I suggest a folder pattern like below are scope the functionality. All the `components`, `tests`, `utils` should be scoped to respective folders. 

And types should be scoped to the file (my preference) or to each folder (if you have shared types) and exported from `index.ts` if needed.  

```
export { default } from './MarketingCollectionCreate';
export * from './MarketingCollectionCreate';
```

<img width="355" alt="Screenshot 2022-06-02 at 18 05 25" src="https://user-images.githubusercontent.com/8556085/171674630-bc753abc-7241-4961-84a2-8c7df6e18ab5.png">

So in parent components can just import `MarketingCollectionCreate` and it will be resolved from `index.ts` and when searched for `MarketingCollectionCreate.tsx` component in Cmd+P in VSCode will get the component.

Reply:
@Mi-Lan In the end this pattern was not agreed with the rest of the team so I'm ok with reverting it. For the rest of the components that I refactored in this manner let's make a separate PR.